test/InterfacesTest.ts

```diff
@@ -50,6 +50,20 @@ const runFunction: AzureFunction = async function(context: Context) {
     return "Ran function";
 }
 
+const cookieFunction: AzureFunction = async function(context: Context) {
+    let cookies: Cookie[] = [
+        {
+            name: "cookiename",
+            value: "cookievalue",
+            expires: Date.now()
+        }
+    ];
+    context.res = {
+        cookies,
```

Comment:
- update test to add multiple cookies
- update test to verify cookies are set on the response.